inference-engine/ie_bridges/python/sample/hello_classification/hello_classification.py

```diff
@@ -30,9 +30,8 @@ def build_argparser():
     args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Show this help message and exit.')
     args.add_argument("-m", "--model", help="Required. Path to an .xml or .onnx file with a trained model.", required=True,
                       type=str)
-    args.add_argument("-i", "--input", help="Required. Path to a folder with images or path to an image files",
-                      required=True,
-                      type=str, nargs="+")
+    args.add_argument("-i", "--input", help="Required. Path to image file.",
```

Comment:
I suppose it should be: Path to ```an``` image file

Reply:
yeh, good catch ) will fix

Reply:
fixed